__tests__/app.test.js

```diff
@@ -80,3 +79,32 @@ describe("/api/articles", () => {
       });
   });
 });
+describe("/api/articles/:article_id", () => {
+  test("GET - 200: responds with an object of the corresponding id ", () => {
+    return request(app)
+      .get("/api/articles/5")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article;
+
+        expect(article).toEqual({
+          article_id: 5,
+          title: "UNCOVERED: catspiracy to bring down democracy",
+          topic: "cats",
+          author: "rogersop",
+          body: "Bastet walks amongst us, and the cats are taking arms!",
+          created_at: "2020-08-03T13:14:00.000Z",
+          votes: 0,
+        });
+      });
+  });
+
+  test("GET - 400: responds with bad request when query unvalid id.", () => {

```

Comment:
this 400 test would be a 404 not found, it's the right type for a review id (a number) that could feasibly exist, a 400 invalid id would be something like /api/articles/bananas. So we'd